### PR TITLE
Fix redis site creation

### PR DESCRIPTION
**Pull request by kirtangajjar**
This PR fixes creation of redis site and also adds tests for redis site creation.

### CODE REVIEW

File: src/Site_WP_Docker.php [hunk 1]
```diff
@@ -83,7 +89,7 @@ public function generate_docker_compose_yml( array $filters = [] ) {
 		// nginx configuration.
 		$nginx['service_name'] = [ 'name' => 'nginx' ];
 		$nginx['image']        = [ 'name' => 'easyengine/nginx:' . $img_versions['easyengine/nginx'] ];
-		$nginx['depends_on']   = [ 'name' => 'php' ];
+		$nginx['depends_on']['dependency'][]   = [ 'name' => 'php' ];
```
Comment (mrrobot47) [inline src/Site_WP_Docker.php]: Fix alignment by running phpcbf.

Comment (kirtangajjar) [reply src/Site_WP_Docker.php]: Done

**Comment by mrrobot47** [comment]
Need to remove caching from `/ee-admin/` url path and solve the below problem:

```^_^[www-data@radhe.dev.rtdemo.in:~/htdocs]$ wp redis
usage: wp redis cli
   or: wp redis debug [--url=<url>] [--format=<format>]
   or: wp redis enable
   or: wp redis info [--reset] [--field=<field>] [--format=<format>]

See 'wp help redis <command>' for more information on a specific command.
^_^[www-data@radhe.dev.rtdemo.in:~/htdocs]$ wp redis info
Error: Warning! PHPRedis module is unavailable, which is required by WP Redis object cache.```

**Comment by mbtamuli** [comment]
https://github.com/EasyEngine/site-wp-command/blob/develop/templates/config/nginx/default.conf.mustache#L25

**Pull request merged**